packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -103,6 +103,30 @@ describe('App: Settings', () => {
       cy.contains('Record Key').should('not.exist')
       cy.contains('Record Key')
     })
+
+    it('clears nested cloud data (record key) upon logging out', () => {
+      cy.startAppServer('e2e')
+      cy.loginUser()
+      cy.visitApp()
+      cy.withCtx((ctx, o) => {
+        o.sinon.spy(ctx.actions.auth, 'logout')
+      })
+
+      cy.get(`[href='#/settings']`).click()
+      cy.contains('Dashboard Settings').click()
+      cy.contains('Record Key').should('exist')
+      cy.get(`[href='#/runs']`).click()
```

Comment:
Do we need to navigate away from the Settings page and log out there so that data is queried again when we return after invalidation? Or could we just log out from the Settings page and see it update immediately after invalidation?

Reply:
Currently before this change, if you're on the settings page it'll send a signal to refetch the mounted queries, the bug is when you're not on the settings page, the queries don't get refreshed.